DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Parts/TitlePartGraphSyncer.cs

```diff
@@ -34,5 +43,23 @@ public override Task AddSyncComponents(JObject content, IGraphMergeContext conte
                 NodeTitlePropertyName,
                 context.NodeWithOutgoingRelationships.SourceNode));
         }
+
+        //we're now doing this for all cloned content items, we happy with that?
+        public override Task MutateOnClone(JObject content, ICloneContext context)
+        {
+            string? title = (string?)content[nameof(TitlePart.Title)];
+
+            if (string.IsNullOrWhiteSpace(title))
+            {
+                throw new InvalidOperationException($"Cannot mutate {nameof(TitlePart)} if {nameof(TitlePart.Title)} is missing.");
```

Comment:
If we allow a content item to have a blank title, we should allow users to clone items with blank titles.
We should just let `_titlePartCloneGenerator.Generate` handle it as normal, so it generates 'CLONE - '.